### PR TITLE
Countries main page

### PR DESCRIPTION
**Pull request by MissMelisa**
Add countries' page and its styles. 
image

### CODE REVIEW

File: src/Components/Countries.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import Card from "@material-ui/core/Card";
+import styles from "./styles.module.css";
+
+function Countries({
+  countryCode,
+  language,
+  countryName,
+  dialCode,
+  spanName,
```
Comment (falecci) [inline src/Components/Countries.jsx:12]: What's this?

File: src/Pages/CountriesPage.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { useQuery } from "react-query";
+
+import Countries from "../Components/Countries";
+import styles from "./styles.module.css";
+import Paper from "@material-ui/core/Button";
+import InputBase from "@material-ui/core/InputBase";
+import IconButton from "@material-ui/core/IconButton";
+import SearchIcon from "@material-ui/icons/Search";
```
Comment (falecci) [inline src/Pages/CountriesPage.jsx:9]: Can you move these to the top?

**Review by falecci — CHANGES_REQUESTED** [review]

File: src/Components/Countries.jsx (hunk 1 above)
Comment (MissMelisa) [reply src/Components/Countries.jsx]: fix it

File: src/Components/Countries.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        <span className={styles.currency}>Currency: {currency}</span>
+        <span className={styles.dialCode}>Int. Dial Code: {dialCode}</span>
+        <span className={styles.language}>Language: {language}</span>
+      </div>
+    </Card>
+  );
+}
+Countries.propTypes = {
+  countryName: PropTypes.string.isRequired,
+  spanName: PropTypes.string.isRequired,
```
Comment (falecci) [inline src/Components/Countries.jsx:32]: I think you forgot to change it here.

File: src/Pages/CountriesPage.jsx (hunk 2 above)
Comment (falecci) [reply src/Pages/CountriesPage.jsx:7]: Thanks!

**Review by falecci — CHANGES_REQUESTED** [review]

**Review by falecci — APPROVED** [review]

**Pull request merged**